app/src/main/java/com/keylesspalace/tusky/components/account/list/ListsForAccountFragment.kt

```diff
@@ -117,7 +123,7 @@ class ListsForAccountFragment : DialogFragment(), Injectable {
             }
         }
 
-        viewLifecycleOwner.lifecycleScope.launch {
+        lifecycleScope.launch {
```

Comment:
Not quite sure what I'm doing with lifecycles here.

With the usage of onCreateDialog() however there seems to never be a point during creation (like "onCreated") where the view is present, so viewLifecycleOwner is always null.

Reply:
If that is the case, I think it is ok to just use `lifecycleScope.launch`